app/controllers/user.js

```diff
@@ -3,6 +3,15 @@ const authenticationService = require('../services/authentication');
 const encryptionService = require('../services/encryption');
 const userService = require('../services/user');
 
+const defineOrder = order => {
+  const ascOrder = 'ASC';
+  const descOrder = 'DESC';
+  if (order && (order.toUpperCase() === ascOrder || order.toUpperCase() === descOrder)) {
```

Comment:
As i said before, this should be checked through express-validator (because is working like a manual validator)